smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpProtocolUnitTestResponseGenerator.kt

```diff
@@ -119,82 +109,12 @@ open class HttpProtocolUnitTestResponseGenerator protected constructor(builder:
     }
 
     protected open fun renderAssertions(test: HttpResponseTestCase, outputShape: Shape) {
-        val nestedShapes = model.getNestedShapes(operation)
-        for (shape in nestedShapes) {
-            when (shape.type) {
-                ShapeType.STRUCTURE -> renderEquatable(shape)
-                ShapeType.UNION -> renderEquatable(shape)
-                else -> {}
-            }
-        }
         writer.write("XCTAssertEqual(actual, expected)")
     }
 
-    private fun identifier(ctx: ProtocolGenerator.GenerationContext, shape: Shape): String {
```

Comment:
All of this rendering code is moved to the SwiftIntegration named `TestEquatableConformanceIntegration` below.